test/misc.js

```diff
@@ -36,4 +36,9 @@ describe('Misc', function() {
     assert.equalIgnoreSpaces(teddy.render('misc/serverSideCommentsNested.html', model), '<p>Any comments? </p>');
     done();
   });
+
+  it('should ignore and skip rendering orphaned argument (misc/orphanedArgument.html)', function(done) {
+    assert.equalIgnoreSpaces(teddy.render('misc/orphanedArgument.html', model), '');
```

Comment:
also i'd move the template to the includes section since it's an `<arg>` tag that's being tested